app/repositories/slick/implementations/SlickChatsRepository.scala

```diff
@@ -670,7 +731,7 @@ class SlickChatsRepository @Inject() (db: Database)(implicit executionContext: E
     (for {
       subject <- ChatsTable.all.filter(_.chatId === chatId).map(_.subject)
       _ <- UserChatsTable.all.filter(userChatRow => userChatRow.chatId === chatId && userChatRow.userId === userId &&
-        (userChatRow.inbox === 1 || userChatRow.sent === 1 || userChatRow.draft === 1 || userChatRow.trash === 1))
+        (userChatRow.inbox === 1 || userChatRow.sent === 1 || userChatRow.draft >= 1 || userChatRow.trash === 1))
```

Comment:
Can draft be bigger than 1?

Reply:
Yes, we are allowing users to have more than one draft per chat